### DISCUSSION TITLE
Can't run my nextjs app in Node server -- Unexpected identifier

### DISCUSSION THREAD

**Discussion by cguidog** [vercel/next.js#10847] [Help]
I'm deploying a next app, but when I try to run it I get the following error:

```
import React, { useState, useEffect } from 'react';
       ^^^^^

SyntaxError: Unexpected identifier
```
It seems as if Node is not identifying the code. I understand that that is why we have to use Babel, so Node can understand ES6 sintax. I don't think I have to add Babel configuration since I didn't have to do that when I was running my NextJS project locally, but I actually tried that anyway and got an error when trying to build my app.

This seems to be an issue related with the Node environment itself and not Next, but I was wondering if somebody has encounter something like this.

I am deploying this site using Plesk, and it seems as if Plesk uses PhusionPassenger to serve the app.

**Comment by cheffley6** [comment]
running into the same error with another project currently. what did you do to resolve this issue?

**Reply by cguidog** [reply]
My problem was that I was using the wrong starter file to build the app. 
I was using "pages/index.js" when it should have been "node_modules/.bin/next"

I hope that helps.
